trnsysGUI/connection/hydraulicExport/singlePipe/createExportHydraulicSinglePipeConnection.py

```diff
@@ -0,0 +1,41 @@
+import dataclasses as _dc
+
+import trnsysGUI.connection.connectorsAndPipesExportHelpers as _helpers
+import trnsysGUI.connection.hydraulicExport.common as _com
+import trnsysGUI.connection.hydraulicExport.singlePipe.singlePipeConnection as _hespc
+import trnsysGUI.massFlowSolver.networkModel as _mfn
+
+
+@_dc.dataclass
+class HydraulicSinglePipeConnection(_com.HydraulicConnectionBase):
+    modelPipe: _mfn.Pipe
+
+
+def createExportHydraulicConnection(
+    hydraulicConnection: HydraulicSinglePipeConnection,
+) -> _hespc.ExportHydraulicSinglePipeConnection:
+    inputTemperature = _helpers.getTemperatureVariableName(
+        hydraulicConnection.fromAdjacentHasPiping.hasInternalPiping,
```

Comment:
Extract and use in analogous file for double pipe. The difference is only in `STANDARD`, `HOT` and `COLD`.